### DISCUSSION TITLE
Can't change SOC password

### DISCUSSION THREAD

**Discussion by hacker0ni** [Security-Onion-Solutions/securityonion#6668] [Unsupported Versions]
Hi there! I've never had an issue with changing the SOC analyst password before. This account is the superuser and I wanted to rotate the password but I'm getting the error message `Unable to save settings: additionalProperties "status" not allowed` on the latest version when trying to change the password on webUI. Any fixes? Thanks!

**Comment by dougburks** [comment]
Here are some things you can try:
- logout, log back in, then try changing your password again
- shift-reload the page in your browser
- try a different browser
